### DISCUSSION TITLE
Please test with Docutils 0.19b1

### DISCUSSION THREAD

**Discussion by AA-Turner** [pradyunsg/furo#463] [Potential Issue] [answered]
Hi,

We've just released Docutils 0.19b1, and intend to release 0.19 final on 05/07/2022 (a week on Tuesday). Please would you test with the pre-release, and raise any issues to me either on this issue or on the Docutils tracker?

- The release notes are at: https://docutils.sourceforge.io/RELEASE-NOTES.html#release-0-19b1-2022-06-21
- The detailed change history is at: https://docutils.sourceforge.io/HISTORY.html#release-0-19b1-2022-06-21
- You can install 0.19b1 with `python -m pip install --pre "docutils==0.19b1"` or install Docutils' master with `python -m pip install "git+https://repo.or.cz/docutils.git#subdirectory=docutils"`.

Thanks,
Adam

**Comment by pradyunsg** [comment, marked as answer]
Will do! :)
